src/cloud-element-templates/util/propertyUtil.js

```diff
@@ -931,3 +929,12 @@ function isOnlyProperty(moddleElement, propertyName) {
     return propertyName === name || moddleElement.get(name) === undefined;
   });
 }
+
+function defaultTranslate(template, replacements) {
```

Comment:
Let's add a TODO here, because I see this as an anti-pattern.

Reply:
How would you fix this?

Reply:
Sketch: https://github.com/bpmn-io/bpmn-js-element-templates/pull/53/commits/8ec997e2bfdefa2c8e58453ce314c6c8d807797d

Reply:
> How would you fix this?

Well, I regard this as a bundler bug. The proper way to use this is to rely on the diagram-js export, not copy paste.

I wonder why it would tree shake like this in the first place. Is the utility not being used in the `core` distribution?

Reply:
I suspect that this is due to `default` export, but can't tell for sure. Let me experiment with a named export...

Reply:
Named export doesn't help here.

Reply:
The thing is that `validateProperty` which uses `translate` is not used in `core` at all, so we should not need that.

Reply:
Yes and yes. That was my suspicion.

> I wonder why it would tree shake like this in the first place. Is the utility not being used in the core distribution?

To be frank I'd be fine to not workaround this issue (unless it causes us troubles elsewhere).

Again, this is a bundler issue.

Reply:
So your final suggestion is to revert the changes and keep it as is?

Reply:
This will cause warnings in the bundler though.

Reply:
> This will cause warnings in the bundler though.

In which bundler? As folks consume it, or as we bundle?

Reply:
As folks consume it via `esbuild`.

Reply:
Namely in `element-templates-cli` project.

Reply:
Let's merge it then.